packages/oauth/oauth-provider/src/lib/util/ui8.ts

```diff
@@ -0,0 +1,14 @@
+export function parseUi8Hex(v: string) {
+  return asUi8(parseInt(v, 16))
+}
+
+export function parseUi8Dec(v: string) {
+  return asUi8(parseInt(v, 10))
+}
+
+export function asUi8(v: number) {
+  if (v >= 0 && v <= 255 && v === (v | 0)) return v
```

Comment:
Is `v === (v | 0)` used instead of `Number.isInteger(v)` as an optimization?

Reply:
This was an optimization of `Nunber.floor(v) === v`, yeah. I didn't know about `Number.isInteger`. Gonna use that instead.